### PR TITLE
Remove only Redash containers

### PR DESCRIPTION
**Pull request by ariarijp**
Hi, team

`make clean` removes all of docker containers for now.

I think `make clean` should be removed only Redash containers.

Thanks,

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,7 +10,7 @@ create_database:
 	docker-compose run server create_db
 
 clean:
-	docker ps -a -q | xargs docker kill;docker ps -a -q | xargs docker rm
+	docker ps -a | grep `basename "$$PWD"` | awk '{print $$1}' | xargs docker kill;docker ps -a | grep `basename "$$PWD"` | awk '{print $$1}' | xargs docker rm
```
Comment (arikfr) [inline Makefile]: Can we use docker-compose commands instead here?

Comment (ariarijp) [reply Makefile]: Oh, what a simple solution!

I gonna fix it.

File: Makefile [hunk 2]
```diff
@@ -17,7 +17,7 @@ create_database:
 	docker-compose run server create_db
 
 clean:
-	docker ps -a -q | xargs docker kill;docker ps -a -q | xargs docker rm
+	docker-compose down && docker-compose rm
```
Comment (ariarijp) [inline Makefile]: @arikfr I've fixed it!

**Comment by arikfr** [comment]
Thanks!

**Pull request merged**